projects/helper/tokenMapping.js

```diff
@@ -112,6 +112,9 @@ const fixBalancesTokens = {
   bsquared: {
     '0x0000000000000000000000000000000000000000': { coingeckoId: "bitcoin", decimals: 18, },
   },
+  bfc: {
+    '0x6906Ccda405926FC3f04240187dd4fAd5DF6d555': { coingeckoId: "usd-coin", decimals: 18, },
```

Comment:
hi @tlatkdgus1 most of the PR is good, except for this part. BtcUSD is not USDC, can wait for coingecko listing before adding it here, another option is, if you share some uni v2/v3 pool with enough liquidity, we can use it to price it in our server.